gt-scheduler/website
# Fix Mobile Course Container Overflow Issue

### Summary

Fixes issue where course container width overflows off the screen on mobile for long course names.

### How to Test
Add a course with a long course name (VIP 3602 is an example) on mobile and check if there are any issues
